.travis.yml

```diff
@@ -33,7 +33,7 @@ jobs:
       before_cache:
         # Save tagged docker images
         - mkdir -p $HOME/docker && docker images -a --filter='dangling=false' --format '{{.Repository}}:{{.Tag}} {{.ID}}' | grep 'centos-with-vsyscall:latest' | xargs -n 2 -t sh -c 'test -e $HOME/docker/$1.tar.gz || docker save $0 | gzip -2 > $HOME/docker/$1.tar.gz'
-    - stage: "Build manylinux images"
```

Comment:
Found a way to get things done in parallel, can you update please ?
```
.......
.......

# use YAML aliases and anchors to avoid duplication in stages
# c.f. https://github.com/travis-ci/travis-ci/issues/8295#issuecomment-454457787
manylinux-build: &manylinux-build
      stage: "Build manylinux images"
      before_install:
        # Load cached docker images
        - if [[ -d $HOME/docker ]]; then ls $HOME/docker/*.tar.gz | xargs -I {file} sh -c "zcat {file} | docker load"; fi
      script:
        - PLATFORM=$PLATFORM TRAVIS_COMMIT=$TRAVIS_COMMIT ./build.sh
      deploy:
        provider: script
        script: docker/deploy.sh
        on:
          branch: master
          repo: pypa/manylinux

jobs:
  include:
    - stage: "Patch glibc"
      env:
        - PLATFORM="x86_64"
      before_install:
        # Load cached docker images
        - if [[ -d $HOME/docker ]]; then ls $HOME/docker/*.tar.gz | xargs -I {file} sh -c "zcat {file} | docker load"; fi
      script:
        - PLATFORM=$PLATFORM TRAVIS_COMMIT=$TRAVIS_COMMIT ./build.sh glibc_only
      before_cache:
        # Save tagged docker images
        - mkdir -p $HOME/docker && docker images -a --filter='dangling=false' --format '{{.Repository}}:{{.Tag}} {{.ID}}' | grep 'centos-with-vsyscall:latest' | xargs -n 2 -t sh -c 'test -e $HOME/docker/$1.tar.gz || docker save $0 | gzip -2 > $HOME/docker/$1.tar.gz'
    - <<: *manylinux-build
      env:
        - PLATFORM="x86_64"
    - <<: *manylinux-build
      env:
        - PLATFORM="i686"
```

Reply:
Looks much better! Thanks!